aiida/backends/tests/work/test_workfunctions.py

```diff
@@ -7,6 +7,7 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
+from aiida.common.exceptions import Exit
```

Comment:
Would it be possible to keep this somewhere in `aiida.work`, I think it only relates to that and it's not really an exception in the sense that no user would ever be asked to catch it, only to raise it (strange, I know)

Reply:
Maybe `work.utils` avoids loading any  ORM.